dftimewolf/lib/processors/gcp_cloud_resource_tree_helper.py

```diff
@@ -272,48 +277,113 @@ def __str__(self) -> str:
     output_format = '{:<20s}|{:<18s}|{:<20s}|{:<25s}|{:<18s}|{:<20s}|{:<25s}|{:<18s}|{:<100s}\n'
 
     # Draw table header
-    output = output + dashes + '\n'
-    output = output + output_format.format(
-        'Resource ID', 'Resource Type', 'Creation TimeStamp', 'Created By',
-        'Creator IP Addr', 'Deletion Timestamp', 'Deleted By',
-        'Deleter IP Addr', 'Tree')
-    output = output + dashes + '\n'
-
+    output = f'{output} {dashes} \n'
+    output = f"""{output} {
+        output_format.format(
```

Comment:
why not just remove the output_format variable and have a single fstring (rather than embedding a format within an fstring)?

Reply:
Done